ESPixelStick/src/input/externalInput.h

```diff
@@ -47,15 +47,15 @@ class c_ExternalInput final
 #   define M_ID         CN_id
 
 	String                    m_name;
-	uint32_t                  m_iPinId              = 0;

```

Comment:
This is done intentionally. I hate the init in the constructor.

Reply:
Ah.  OK.  I'll revert initialization back to the header...

I will also add your strong preference to the project style guide.

Reply:
Done.

Reply:
Done.